### ISSUE TITLE
[DOC] Rework tutorials

### ISSUE THREAD

**Issue by mloning** [alan-turing-institute/sktime#1246]

**Comment by fkiraly** [comment]
well, hopefully you don't think the forecasting one is in need of this? I thought it was in a good state.
Classification definitely, I think this needs more structure and more pipeline/composition examples.

**Comment by RNKuhns** [comment]
@fkiraly I think that the idea is to make things more streamlined. Carrying around the output of cells generally isn't what we want to track on Github. And if we are expecting the notebooks to be run when we generate the docs we need them to run more quickly. So the focus would be demonstrating techniques with algorithms that run quickly and pointing users to User Guides and API reference for other estimators that could be used in the same data science task. 

Demonstrations of certain estimators (e.g. How to use ROCKET, but also specific forecasters) would be how-to-guides in .py files that we'd ideally include as "tiles" in the web documentation (like those code examples on scikit-learn).

**Comment by fkiraly** [comment]
@RNKuhns, agreed - tutorials should run quickly, and output should not be carried around.
